reference/clever-tools/getting_started.md

```diff
@@ -34,12 +34,18 @@ npm install -g clever-tools@beta
 
 #### Debian/Ubuntu (.deb)
 
+{{< alert "warning" "End of Bintray" >}}
+<p>Bintray is ending on 1st May, we will migrate hosting as soon as possible.</p>
```

Comment:
You should specify the year here. And the date should be written `May 1st 2021.`